docs/md/checkpoint_learn_build.md

```diff
@@ -69,7 +75,7 @@ With these set, invoke the script with two arguments: the path to the
 $ checkpoint/ci/build_cpp.sh <full-path-to-checkpoint-source> <full-path-to-build-dir>
 ```
 
-\subsection docker-build Building with `docker` containerization
+\subsection docker-build Building with docker containerization
```

Comment:
It fixes this:
![image](https://user-images.githubusercontent.com/14062470/116563899-c54c3000-a904-11eb-90b7-e652b30f84b8.png)